unyt/array.py

```diff
@@ -246,14 +241,6 @@ def _unit_repr_check_same(my_units, other_units):
     is compatible with this quantity. Returns Unit object.
 
     """
-    equiv_dims = em_dimensions.get(my_units.dimensions, None)
```

Comment:
I assume we never get here anymore, correct?

Reply:
yup, this section is now handled by `_check_em_conversion`